test/milton/MiltonEvents.t.sol

```diff
@@ -0,0 +1,525 @@
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(daiMockedToken), TestConstants.PERCENTAGE_5_18DEC, block.timestamp); // 5%, PERCENTAGE_5_18DEC
+        vm.prank(_liquidityProvider);
+        josephDai.itfProvideLiquidity(TestConstants.USD_28_000_18DEC, block.timestamp); // TestConstants.USD_28_000_18DEC
+        vm.prank(_userTwo);
+        miltonDai.itfOpenSwapPayFixed(
+            block.timestamp, // openTimestamp
+            TestConstants.USD_10_000_18DEC, // totalAmount
+            TestConstants.PERCENTAGE_6_18DEC, // acceptableFixedInterestRate, 6%
+            TestConstants.LEVERAGE_18DEC // leverage
+        );
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(daiMockedToken), TestConstants.PERCENTAGE_160_18DEC, block.timestamp); // PERCENTAGE_160_18DEC
+        vm.prank(_userTwo);
+        vm.expectEmit(true, true, true, true);
+        emit CloseSwap(
+            1, // swapId
+            address(daiMockedToken), // asset
+            block.timestamp + TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS, // closeTimestamp, 28 days, PERIOD_28_DAYS_IN_SECONDS
+            _userTwo, // liquidator
+            18957318804358692392282, // transferredToBuyer
+            TestConstants.ZERO, // transferredToLiquidator
+            TestConstants.TC_INCOME_TAX_18DEC // incomeFeeValue
+        );
+        miltonDai.itfCloseSwapPayFixed(
+            1, // swapId
+            block.timestamp + TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS // closeTimestamp, 28 days, PERIOD_28_DAYS_IN_SECONDS
+        );
+    }
+
+    function testShouldEmitEventWhenClosePayFixedSwap6DecimalsAndTakerClosedSwap() public {
+        // given
+        MockTestnetToken usdtMockedToken = getTokenUsdt();
+        ItfIporOracle iporOracle = getIporOracleAsset(_userOne, address(usdtMockedToken), 0);
+        IpToken ipTokenUsdt = getIpTokenUsdt(address(usdtMockedToken));
+        MockCase0Stanley stanleyUsdt = getMockCase0Stanley(address(usdtMockedToken));
+        MiltonStorage miltonStorageUsdt = getMiltonStorage();
+        MockCase0MiltonUsdt miltonUsdt = getMockCase0MiltonUsdt(
+            address(usdtMockedToken),
+            address(iporOracle),
+            address(miltonStorageUsdt),
+            address(_miltonSpreadModel),
+            address(stanleyUsdt)
+        );
+        MockCase0JosephUsdt josephUsdt = getMockCase0JosephUsdt(
+            address(usdtMockedToken),
+            address(ipTokenUsdt),
+            address(miltonUsdt),
+            address(miltonStorageUsdt),
+            address(stanleyUsdt)
+        );
+        prepareApproveForUsersUsd(_users, usdtMockedToken, address(josephUsdt), address(miltonUsdt));
+        prepareMilton(miltonUsdt, address(josephUsdt), address(stanleyUsdt));
+        prepareJoseph(josephUsdt);
+        prepareIpToken(ipTokenUsdt, address(josephUsdt));
+        // when
+        _miltonSpreadModel.setCalculateQuotePayFixed(TestConstants.PERCENTAGE_6_18DEC); // 6%
```

Comment:
If you are doing copy paste in tests please do it in the same order "setCalculateQuotePayFixed" was at the beginning.